test/tests/misc/check_error/bad_parsed_function_vars.i

```diff
@@ -17,6 +17,7 @@
     type = ParsedFunction
     value = sin(y)
     vars = y        # <- This is a bad - you can't specify x, y, z, or t
+    vals = 0
```

Comment:
You could get your coverage up by one line by creating one additional test with and without this `vals = 0` line.